src/stories/theme/themes.ts

```diff
@@ -501,7 +798,7 @@ export const LightTheme: Theme = {
         },
 
         dropdown: {
-            shadow: Shadows.deepBelow,
+            shadow: '0px 8px 20px 0px rgba(0, 0, 0, 0.2)',
```

Comment:
That shadow has different light mode and dark mode variants, hence me not just defining all shadows together?

Reply:
I guess also thought that uses of borders-dropdown-shadow should probably be replaced with one of the shadow variables directly, one so they can change for dark/light mode, and two so that everything can reference the defined Figma styles. So this value could end up being removed when the components are refactored anyway?

Reply:
see https://github.com/slingshot/paris/pull/44#discussion_r1747798368

Reply:
> I guess also thought that uses of borders-dropdown-shadow should probably be replaced with one of the shadow variables directly, one so they can change for dark/light mode, and two so that everything can reference the defined Figma styles. So this value could end up being removed when the components are refactored anyway?

yeah, this is fair. I think the original idea here was that we had a number of dropdown surfaces across various types of components using this and we wanted to keep them all mapped to one abstracted variable, but if we define it directly in Figma anyways then it could be fine to match that approach